### PR TITLE
Doc add sha to footer

### PR DESCRIPTION
**Pull request by tacaswell**
## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is PEP 8 compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -138,14 +141,14 @@ require changes to your existing code is logged in the :doc:`API changes
 <api/api_changes>` file.
 
 Toolkits
-========
+--------
```
Comment (jklymak) [inline doc/index.rst]: I preferred how it looked as level-1...

https://5220-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/index.html
versus
https://matplotlib.org

Comment (dstansby) [reply doc/index.rst]: Agreed, I think https://matplotlib.org/devdocs/ looks fine

Comment (tacaswell) [reply doc/index.rst]: The issue that this is fixing the browser title which shows up as "Installation - Matplotlib 2.xxxxx documentation" vs "Matplotlib - Matplotlib 2.xxxx documentation".

I can seperate these two things, but we should fix the page title (as the browser sees it in `<head><title>Title<\title><\head>`)

Comment (QuLogic) [reply doc/index.rst]: The title is now fixed by #10870.

**Comment by tacaswell** [comment]
force-pushed the index.rst changes out of existence.

**Pull request merged**